dask/dataframe/io/parquet/core.py

```diff
@@ -556,45 +556,60 @@ def apply_filters(parts, statistics, filters):
         Tokens corresponding to row groups to read in the future
     statistics: List[dict]
         List of statistics for each part, including min and max values
-    filters: List[Tuple[str, str, Any]]
-        List like [('x', '>', 5), ('y', '==', 'Alice')]
+    filters: Union[List[Tuple[str, str, Any]], List[List[Tuple[str, str, Any]]]]
+        List of filters to apply, like [[('x', '=', 0), ...], ...]
 
     Returns
     -------
     parts, statistics: the same as the input, but possibly a subset
     """
-    for column, operator, value in filters:
-        out_parts = []
-        out_statistics = []
-        for part, stats in zip(parts, statistics):
-            if "filter" in stats and stats["filter"]:
-                continue  # Filtered by engine
-            try:
-                c = toolz.groupby("name", stats["columns"])[column][0]
-                min = c["min"]
-                max = c["max"]
-            except KeyError:
-                out_parts.append(part)
-                out_statistics.append(stats)
-            else:
-                if (
-                    operator == "=="
-                    and min <= value <= max
-                    or operator == "<"
-                    and min < value
-                    or operator == "<="
-                    and min <= value
-                    or operator == ">"
-                    and max > value
-                    or operator == ">="
-                    and max >= value
-                ):
+
+    def apply_conjunction(parts, statistics, conjunction):
+        for column, operator, value in conjunction:
+            out_parts = []
+            out_statistics = []
+            for part, stats in zip(parts, statistics):
+                if "filter" in stats and stats["filter"]:
+                    continue  # Filtered by engine
+                try:
+                    c = toolz.groupby("name", stats["columns"])[column][0]
+                    min = c["min"]
+                    max = c["max"]
+                except KeyError:
                     out_parts.append(part)
                     out_statistics.append(stats)
+                else:
+                    if (
+                        operator == "=="
+                        and min <= value <= max
+                        or operator == "<"
+                        and min < value
+                        or operator == "<="
+                        and min <= value
+                        or operator == ">"
+                        and max > value
+                        or operator == ">="
+                        and max >= value
+                        or operator == "in"
```

Comment:
Is `"in"` supported in pyarrow?  It certainly makes sense to me - Just curious.

Reply:
Yes, it is: https://github.com/apache/arrow/blob/ccff05b9f9f388530c7a4db50870dc3b7decc838/python/pyarrow/parquet.py#L788